templates/statfile.html

```diff
@@ -86,6 +86,34 @@
                     {{ meetcont | safe }}
                 </div>
                 <div class="card-footer text-center text-white" style="background-color: #008080;">
+                    <div class="footer-top">
+                        <div class="about">
+                            <div class="footer-heading"><h5>ABOUT</h5></div>
+                            <div class="footer-link-item"><a href="">Get Fragment</a></div>
+                            <div class="footer-link-item"><a href="">Sponsors</a></div>
+                            <div class="footer-link-item"><a href="">Fragment Magazine</a></div>
+                            <div class="footer-link-item"><a href="">Legal</a></div>
+                        </div>
+                        <div class="support">
+                            <div class="footer-heading"><h5>SUPPORT</h5></div>
+                            <div class="footer-link-item"><a href="">Get Help</a></div>
+                            <div class="footer-link-item"><a href="">Ask Fragment</a></div>
+                            <div class="footer-link-item"><a href="">Common Bugs</a></div>
+                            <div class="footer-link-item"><a href="">Fragment Developer Portal</a></div>
+                        </div>
+                        <div class="community">
+                            <div class="footer-heading"><h5>COMMUNITY</h5></div>
+                            <div class="footer-link-item"><a href="">Join Fragment</a></div>
+                            <div class="footer-link-item"><a href="">About Fragment</a></div>
+                            <div class="footer-link-item"><a href="">Planet Fragment</a></div>
+                            <div class="footer-link-item"><a href="">Fragment Account System</a></div>
+                        </div>
+                    </div>
+                    <div class="footer-mid">
+                        <p class="footer-link-item">
+                            Direct comments and corrections to the<a href="" >websites team</a>.
+                        </p>
+                    </div>
```

Comment:
I would suggest adding these links to the page footer instead of the card footer.